### DISCUSSION TITLE
[Package] Geospatial

### DISCUSSION THREAD

**Discussion by kyleaton** [great-expectations/great_expectations#4367] [Expectations]
# What Is Geospatial Data?

Geospatial data is information that describes objects, events, or phenomena with a location on or near the surface of the earth. It typically combines location information (usually in the form of coordinates) with attribute information (characteristics) and temporal information (denoting time or life span).

## How Can Great Expectations Contribute to Geospatial Data?	

Geospatial data is most useful when it can be analyzed, shared, and integrated with other types of business data to build data visualizations. These visualizations, such as maps, graphs, cartograms, etc., show a more complete picture of the objects, events, or phenomena. 

However, geospatial data is highly prone to error, as it usually involves robust datasets with data gathered from multiple sources in varying formats. This can cause multiple issues down the line. Before the data is used to create interactive maps and visualizations, it needs to be validated.

Linked below are a number of Expectations on Geospatial Data that have already been contributed to Great Expectations!

**Comment by austiezr** [comment]
- [Expect Column Values To Be A Point Within Geo Region](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_values_point_within_geo_region.py)
- [Expect Column Average To Be Point Within Range Of Given Point](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_average_to_be_within_range_of_given_point.py)
- [Expect Column Values To Be US Zipcode Within Radius Of Given Zipcode](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_us_zipcode_within_radius_of_given_zipcode.py)
- [Expect Column Values To Be Lat Lon Coordinates In Range Of Given Point](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py)
- [Expect Column Values To Be Valid Degree-Decimal Coordinates](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_degree_decimal_coordinates.py)

**Comment by luismdiaz01** [comment]
Hey @austiezr wanted to flag this when running some of my negative test on geospatial expectations. 

![Screenshot 2022-04-06 161708](https://user-images.githubusercontent.com/43221010/162089146-c2cea429-34a5-4cc6-8c10-39ef226e70f6.png)
